dev/tools/generate-release-changelog.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+if [ $# != 1 ]; then
+  echo "Usage: $0 BRANCH"
+  exit
+fi
+
+BRANCH=$1
+
+git checkout $BRANCH
+git pull
+CHANGELOG_ENTRIES_WITH_TITLE=$(ls doc/changelog/*/*.rst)
+CHANGELOG_ENTRIES_NO_TITLE=$(echo "$CHANGELOG_ENTRIES_WITH_TITLE" | grep -v "00000-title.rst")
```

Comment:
```suggestion
CHANGELOG_ENTRIES_WITH_TITLE=$(echo doc/changelog/*/*.rst)
CHANGELOG_ENTRIES_NO_TITLE=${CHANGELOG_ENTRIES_WITH_TITLE/00000-title.rst/}
```
should work without using `ls`